pypiserver/values.yaml

```
@@ -70,6 +70,13 @@ persistence:
   ## than one replica
   accessMode: ReadWriteOnce
   size: 5Gi
+  # mountPropagation: HostToContainer
```

The variable needs to be defined:
```suggestion
  mountPropagation: None
```